### DISCUSSION TITLE
[Request] Update `libcpufeatures-static` package

### DISCUSSION THREAD

**Discussion by T-3B** [termux/termux-packages#12453] [Q&A] [answered]
Hi there !

I did not open an issue, since my request is not a new package demand nor a bug.
I want to download the package `libcpufeatures-static`, and its version is `0.7.0`. The problem is that it depends on the package `libcpufeatures`, with a version of `0.7.0-1`. This has been like that for many days.

There is therefore a problem, and it is currently _impossible_ to install the static package.

```
The following packages have unmet dependencies:
 libcpufeatures-static : Depends: libcpufeatures (= 0.7.0) but 0.7.0-1 is to be installed
E: Unable to correct problems, you have held broken packages.
```

```
libcpufeatures/stable,now 0.7.0-1 aarch64 [installed]

libcpufeatures-static/stable 0.7.0 aarch64
```

```
E: Version '0.7.0' for 'libcpufeatures' was not found
```

Android 11 - aarch64.

**Comment by truboxl** [comment, marked as answer]
Hi, this is a bug. Seems like libcpufeatures-static package didn't get removed
You should use libcpufeatures from now on since the static files are included there
CC @Grimler91 @licy183

**Reply by Grimler91** [reply]
Removed manually now (will be gone after next package update), thanks

**Comment by licy183** [comment]
Yeah, this is a bug. Actually, there have been many issues due to the lack of ability of our CI. Maybe we should enhance it to make it possible to delete packages.

**Comment by T-3B** [comment]
Oh ok, thanks for your fast replies !

`libcpufeatures` works as expected.

Have a nice day.
